src/crypto/test_multi_party_schnorr.rs

```diff
@@ -175,3 +175,95 @@ pub fn keygen_t_n_parties(
 
     (party_keys_vec, shared_keys_vec, y_sum, vss_scheme_vec)
 }
+
+use curv::arithmetic::traits::Converter;
+use curv::elliptic::curves::secp256_k1::Secp256k1Scalar;
+use curv::elliptic::curves::traits::{ECPoint, ECScalar};
+use curv::BigInt;
+use std::borrow::Borrow;
+
+const STR_SECRET1: &str = "12b004fff7f4b69ef8650e767f18f11ede158148b425660723b9f9a66e61f747";
+const STR_SECRET2: &str = "b524c28b61c9b2c49b2c7dd4c2d75887abb78768c054bd7c01af4029f6c0d117";
+const STR_SECRET1C: &str = "12b004fff7f4b69ef8650e767f18f11ede158148b425660723b9f9a66e61f747";
+const STR_SECRET2C: &str = "b524c28b61c9b2c49b2c7dd4c2d75887abb78768c054bd7c01af4029f6c0d117";
+
+fn get_shared_keys(wif: &str) -> SharedKeys {
```

Comment:
- Rename parameter `wif`, because it is not wif format but hex format.
- Use `wif` parameter in this function. this function always returns SharedKey generated from `STR_SECRET1`

```
warning: unused variable: `wif`
   --> src/crypto/test_multi_party_schnorr.rs:190:20
    |
190 | fn get_shared_keys(wif: &str) -> SharedKeys {
    |                    ^^^ help: consider prefixing with an underscore: `_wif`
    |
    = note: `#[warn(unused_variables)]` on by default
```

Reply:
i will fix.